keda/templates/10-keda-clusterrole.yaml

```diff
@@ -81,6 +81,8 @@ rules:
 - apiGroups:
   - keda.sh
   resources:
```

Comment:
@zroubalik I think this is fine, thoughts?

If so, would you mind sending a PR to https://github.com/kedacore/keda/blob/main/config/rbac/role.yaml#L76 @mlushpenko ?

Reply:
Since we are renaming a resource, there might be problem when doing update of KEDA 2.1 -> 2.2 (or the version that will include this change). Because I am not sure that the "old" resource will be properly deleted? How does Helm handle this?

Reply:
It does upserts AFAIK so that should be OK. Should we port this back to the core?

Reply:
OK! We should add a note to the Release Notes about this change. (Users who install through yaml manifest need to delete the old ClusterRole manually.

Reply:
OMG, sorry please disregard my previous comments. I thought this PR is about renaming the ClusterRole.

This change is absolutely OKay, and should be backported to core

Reply:
> https://github.com/kedacore/keda/blob/main/config/rbac/role.yaml#L76

@tomkerkhove  is this good https://github.com/kedacore/keda/pull/1645?

Reply:
Certainly, thanks!